File: Sprint1/JAVA/Assignments/HitarthPatel_150096724046_31JAN2025_Java_Assignemnt_4/Rectangle.java
import java.util.Scanner;

public class Rectangle {
    private float length;
    private float width;

    public Rectangle() {
        this.length = 1.0f;
        this.width = 1.0f;
    }

    public Rectangle(float length, float width) {
        this.length = length;
        this.width = width;
    }

    public float getLength() {
        return this.length;
    }

    public float getWidth() {
        return this.width;
    }

    public double calculateArea() {
        return this.length * this.width;
    }

    public double calculatePerimeter() {
        return 2 * (this.length + this.width);
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the length of the rectangle (default is 1.0): ");
        float length = scanner.nextFloat();
        System.out.println("Enter the width of the rectangle (default is 1.0): ");
        float width = scanner.nextFloat();
        Rectangle rectangle = new Rectangle(length, width);
        System.out.println("Length: " + rectangle.getLength());
        System.out.println("Width: " + rectangle.getWidth());
        System.out.println("Area: " + rectangle.calculateArea());
        System.out.println("Perimeter: " + rectangle.calculatePerimeter());
        scanner.close();
    }
}

File: Sprint1/DSA/ClassAssignment/20Feb2025/Rectangle.cpp
#include <iostream>
using namespace std;

struct Rectangle{
    float length;
    float width;

};
void Area(Rectangle a){
   cout<<"Area of the Rectangle is :-"<<a.length * a.width<<endl;

}
int main(){

    Rectangle b;
    cout<<"Enter the length of the Rectangle :-"<<endl;
    cin>>b.length;
    cout<<"Enter the width of the Rectangle :-"<<endl;
    cin>>b.width;

    Area(b);

}